src/main/java/Entities/GroupProfile.java

```diff
@@ -0,0 +1,33 @@
+package Entities;
+import java.util.HashMap;
+
+public class GroupProfile {
```

Comment:
1. The entity looks good! Glad you implemented the courseCode attribute into the GroupProfile entity and moved that responsibility from the Group entity as we discussed in tutorial. I think it makes a lot more sense since the GroupProfile is responsible for these additional details.
2. Since GroupProfile is going to be an attribute of Group, this entity should implement Serializable class so that it doesn't cause an error when I serialize the Group entity.

Reply:
Hi yes I will change course code to be an attribute since I think this is easier for you to implement some of your methods @helenaglow. Julia and I also talked about making the profiles an interface but since we only have getters and setters as our methods for our entities it seemed kind of pointless to make a shared interface just for these methods.